Update doc of multi-cluster setup for ICP (#3619)
istio/istio.io: content/docs/examples/multicluster/icp/index.md

1.  Add IP routers from `cluster-2` to `cluster-1`.

{{< warning >}}

> This works if all the nodes among the multiple IBM Cloud Private clusters are located in the same subnet, However, it isn't able to add BGP routers directly for the nodes that are located in different subnets, because the IP addresses cannot be reached in just a hop. In that case, the solution is setting up a VPN for pod communication across clusters, please refer to [this article](https://medium.com/ibm-cloud/setup-pop-to-pod-communication-across-ibm-cloud-private-clusters-add0b079ebf3) for more details.

Reviewer: ```suggestion
This approach works if all the nodes within the multiple IBM Cloud Private clusters are located in the same subnet. It is unable to add BGP routers directly for nodes located in different subnets because the IP addresses must be reachable with a single hop. Alternatively, you can use a VPN for pod communication across clusters. Refer to [this article](https://medium.com/ibm-cloud/setup-pop-to-pod-communication-across-ibm-cloud-private-clusters-add0b079ebf3) for more details.
```

The following example enables [automatic sidecar injection](/docs/setup/kubernetes/additional-setup/sidecar-injection/#automatic-sidecar-injection).

> 1.  Install `bookinfo` on the first cluster `cluster-1`. Remove `reviews-v3` deployment to deploy on remote cluster `cluster-2`:

Reviewer: ```suggestion
1.  Install `bookinfo` on the first cluster `cluster-1`. Remove the `reviews-v3` deployment which will be deployed on cluster `cluster-2` in the following step:
```

Reviewer 2: ```suggestion
1.  Install the `bookinfo` app on the `cluster-1` cluster. Remove the `reviews-v3` service since the next step is to deploy it on the `cluster-2` cluster:
```

    {{< text bash >}}
    $ kubectl apply -f @samples/bookinfo/platform/kube/bookinfo.yaml@
    $ kubectl apply -f @samples/bookinfo/networking/bookinfo-gateway.yaml@
    $ kubectl delete deployment reviews-v3
    {{< /text >}}

> 1.  Deploy the `reviews-v3` deployment and corresponding services on the remote cluster `cluster-2`:

Reviewer 2: ```suggestion
1.  Deploy the `reviews-v3` service along with any corresponding services on the remote `cluster-2` cluster:
```
